ctapipe/instrument/camera.py

```diff
@@ -202,7 +246,7 @@ def _calc_pixel_area(pix_x, pix_y, pix_type):
         Note this will not work on cameras with varying pixel sizes.
         """
 
-        dist = np.min(np.sqrt((pix_x - pix_x[0])**2 + (pix_y - pix_y[0])**2))
+        dist = np.min(np.sqrt((pix_x[1:] - pix_x[0])**2 + (pix_y[1:] - pix_y[0])**2))
```

Comment:
Ah btw. I think this was a bug ... I have no idea why this worked ... I think this line tries to find the distance between the pixels, simply by calculating the distance from pixel zero to all other pixels and then finding the minimum.

The result should be the typical pixel pitch, unless pixel zero happens to sit in a special location outside of the normal grid. However, if one does not exclude pixel zero from the others .. the result should always be zero, since pixel zeros distance from itself is zero, and that should be the minimum, no? 

So I have no idea how this line worked ...